centml/sdk/api.py

```diff
@@ -134,6 +134,29 @@ def get_deployment_usage(
             step=step,
         ).values
 
+    def initialize_user(self):
+        return self._api.setup_stripe_customer_payments_setup_post()
```

Comment:
we can keep these two lines. makes it easy for us to setup the user through the CLI, by explicitly calling cclient.initialize_user()